### ISSUE TITLE
UnifiedJedis is missing support for transactions

### ISSUE THREAD

**Issue by njibhu** [redis/jedis#2857]
## Description

The new `UnifiedJedis` class added with Jedis 4.0.0 is missing support for the `watch` and `multi` commands.
I also couldn't find a way to easily instantiate from the `Transaction` class directly from a UnifiedJedis instance.

**Comment by njibhu** [comment]
@sazzad16 Thanks for the explaination !
Since the single connection client Jedis doesn't support json commands but I need these, I don't have a choice but to use the UnifiedJedis class even though I have a single server.

What would actually be the recommended way to use directly Transaction ? Is there a way to get a single connection from a UnifiedJedis to pass to the Transaction constructor ? Or should I have a normal Jedis instance only to create Transactions and keep the UnifiedJedis for my other queries ?

**Comment by sazzad16** [comment]
@njibhu You can create a Connection and use that to create Jedis, UnifiedJedis, Transaction, etc.

```
try (Connection connection = new Connection(...)) {
  Jedis jedis = new Jedis(connection);
  UnifiedJedis ujedis = new UnifiedJedis(connection);
  Transaction transaction = new Transaction(connection);
}
```

PS: This is not something that I would confidently recommend for production.

**Comment by sazzad16** [comment]
Merged #3361
